hq/markdown/vulnerability-management.md

```diff
@@ -0,0 +1,110 @@
+library will at some point need updating to fix a security concern. Most of the time this should be as simple as changing
+a version number in your build.sbt or package.json file, but it could involve days of work upgrading, for example, your
+version of the Play! Framework. The (draft) standard for Patch Management at The Guardian states:
+
+ ```
+All GNM managed third party applications must be patched on a regular basis as patches are released.
+
+Technical owners are responsible to establish such a process.
+```  
+### 1. Find the vulnerabilities
+At The Guardian we recommend 2 tools for vulnerabilities: Snyk and Dependabot
+
+#### Snyk
+The Guardian has an enterprise account with Snyk, which will scan your repo for vulnerabilities and open PRs where possible
+to resolve them. This is the recommended tool for Scala projects. We have unlimited scans with Snyk so you shouldn't
+hesitate to enable it in your project. Full documentation exists [here](./snyk.md). For a summary of all Snyk vulnerabilities
+at The Guardian in one place you can head to the [Snyk dashboard on Security HQ](https://security-hq.gutools.co.uk/snyk) 
+(VPN required).
+
+#### Dependabot
+Dependabot is a similar tool by Github. For TypeScript/Javascript projects you might choose to use this instead of Snyk
+as it integrates well with the Github UI. (It's not worth switching if you're already set up with Snyk though). It opens
+ automated PRs, but you can also see more detail, including any PRs that have been ignored/closed in the 'Security' tab
+ of your repo. One downside of Dependabot is that there's no API or central dashboard, so the only way of checking dependency
+ issues is by going to each repo one by one.
+ 
+One thing to note is that dependabot will tell you when there's a new version of a library even if there isn't a security
+concern. If you're trying to save time you might opt to skip these - any security related fixes will say so in the PR
+description.  
+ 
+### 2. Prioritise the vulnerabilities
+Soo, your team should have some kind of process for staying up to date with patching. Not every vulnerability highlighted by
+Snyk or Dependabot necessarily needs fixing - maybe the way your app works (e.g. in a private VPC) means that the vulnerability
+listed isn't of concern. However, in many cases it's easier to fix rather than to justify not fixing something though, and in general
+it's good to have 'defence in depth' - so to assume that any other security measures might get compromised and not to rely on
+e.g. network security. 
+
+Initially you might choose to target only 'high' criticality vulnerabilities. 
+
+### 3. Fix them!
+
+This can be the sloow, hard part! In future, there might be links to docs on dealing with e.g. awkward play upgrades,
+dodgy AWS SDK dependencies, but for now all we can say is 
+
+![good luck](https://media2.giphy.com/media/3oeSAz6FqXCKuNFX6o/giphy.gif)
```

Comment:
Would prefer that we avoid hotlinking to sites outside our control, although I appreciate the importance of good gifs. If we want to keep it maybe we can add it as an asset?

sorry!